Calling/ClientApp/src/Utils/Utils.ts

```diff
@@ -77,6 +77,20 @@ export const utils = {
       return { message: 'Recording could not be stopped' };
     }
   },
+  getRecordingLink: async (id: string): Promise<string> => {
+    try {
+      const response = await fetch('/recording/getRecordingLink?serverCallId=' + id);
+      if (response.ok) {
+        const content = await response.json();
+        return content.uri;
+      } else {
+        return '';
```

Comment:
what is the best practices here if we can't get a response from the server?

Reply:
I believe we could use response code from server side, and then add our own error message in the response